openpgp/packet/signature.go

```diff
@@ -331,7 +331,7 @@ func parseSignatureSubpacket(sig *Signature, subpacket []byte, isHashed bool) (r
 		sig.PreferredCompression = make([]byte, len(subpacket))
 		copy(sig.PreferredCompression, subpacket)
 	case primaryUserIdSubpacket:
-		// Primary User ID, section 5.2.3.19
+		// Primary User Id, section 5.2.3.19
```

Comment:
But then here in text, I think it's fine to write it normally, so in both caps, as it's not referring to the property's name. (In signature_v3.go as well.)

Reply:
Okay, sticking to that +1